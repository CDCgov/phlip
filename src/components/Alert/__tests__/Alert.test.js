import React from 'react'
import { shallow } from 'enzyme'
import { Alert } from '../index'

const props = {
  open: true,
  title: 'Test alert title',
  actions: [
    { value: 'Cancel', type: 'button', onClick: () => {} },
    { value: 'Continue', type: 'button', onClick: () => {} }
  ],
  children: <div>I am the alert content.</div>
}

describe('Alert component', () => {
  test('should render correctly', () => {
    expect(shallow(<Alert {...props} />)).toMatchSnapshot()
  })
})