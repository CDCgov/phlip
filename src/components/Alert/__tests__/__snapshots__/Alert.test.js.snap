// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert component should render correctly 1`] = `
<WithStyles
  open={true}
>
  <ModalTitle
    closeButton={false}
    edit={false}
    editButton={false}
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
      }
    }
    title="Test alert title"
  />
  <ModalContent
    style={
      Object {
        "minWidth": 350,
      }
    }
  >
    <div>
      I am the alert content.
    </div>
  </ModalContent>
  <WithStyles
    actions={
      Array [
        Object {
          "onClick": [Function],
          "type": "button",
          "value": "Cancel",
        },
        Object {
          "onClick": [Function],
          "type": "button",
          "value": "Continue",
        },
      ]
    }
  />
</WithStyles>
`;
